Dates specified to be included or excluded (#177)
MewsSystems/gitbook-channel-manager-api: channel-manager-operations/availabilityBlock.md

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |

> | `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2021-12-24"`\).  |

Reviewer: ```suggestion
| `start` | `string` | required | Inclusive start date of the block, in format `yyyy-mm-dd`, e.g. `2021-12-24`; this is the first day of the block and so can be booked for stay.  |
```

> | `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2021-12-31"`\). This date can't be booked for stay anymore. This date is like a reservation departure. |

Reviewer: ```suggestion
| `end` | `string` | required | Exclusive end date of the block, in format `yyyy-mm-dd`, e.g. `2021-12-31`; this day is _not_ included in the block and so cannot be booked for stay (it is analogous to reservation departure). |
```

> | `releasedDate` | `string` | required | End date in format `"yyyy-MM-dd"` \(e.g., `"2021-12-24"`\). |

Reviewer: I would like to update this field as well to follow the same format, but I note the current description just says "End date", what exactly is the release date?

> > **Dates**: Interval for 1 night (e.g. is represented 2021-12-24/25 is represented as `"start": "2021-12-24", "end": "2021-12-24"`).

Reviewer: I would prefer to avoid additional explanations, because in future this API will likely go to semi-automated documentation, in which case it won't be possible to add additional explanatory text in the middle of the API Reference.

In any case, unless I'm mistaken, the example is wrong! Since the end date is exclusive, like a reservation departure, then surely for a 1 night stay it should be start "2021-12-24" and end "2021-12-25", or am I mistaken?
```suggestion
```

Author: right 🤦
